src/js/components/PlacementConstraintsPartial.js

```diff
@@ -152,27 +129,27 @@ export default class PlacementSection extends Component {
               placeholder="hostname"
               value={constraint.fieldName}
             />
+            <FieldHelp>
+              E.g hostname.
+            </FieldHelp>
             <FieldError>{fieldNameError}</FieldError>
           </FormGroup>
           <FormGroup
-            className={{
-              "column-4": !hideValueColumn,
-              hidden: hideValueColumn
-            }}
+            className={commonFieldsClassNames}
             required={valueIsRequired}
             showError={Boolean(valueError)}
           >
             {valueLabel}
-            <FieldInput
-              className={{ hidden: valueIsRequiredEmpty }}
-              name={`constraints.${index}.value`}
-              type="text"
-              value={constraint.value}
-            />
-            <FieldHelp
-              className={{ hidden: valueIsRequired || valueIsRequiredEmpty }}
-            >
-              This field is optional
+            {valueIsRequiredEmpty
+              ? this.getToolTip(
+                  fieldValue,
+                  OperatorTypes[constraint.operator].tooltipContent
+                )
+              : fieldValue}
+            <FieldHelp>
+              {OperatorTypes[constraint.operator]
```

Comment:
related to my comment above, shouldnt it be possible to write something like this?
```
{OperatorTypes[constraint.operator].requiresValue ||
  OperatorTypes[constraint.operator].requiresValueEmpty 
  ? "A string, integer or regex value." 
  : "A string, integer or regex value. This field is optional."
}
```
maybe we can also include this part into `fieldValue`